### PR TITLE
Cleaned up copy methods

### PR DESCRIPTION
**Pull request by davidpiuva**
Removing some half finished deprecated code.

### CODE REVIEW

File: source/draw/RasterImage.ooc [hunk 1]
```diff
@@ -22,33 +22,10 @@ RasterImage: abstract class extends Image {
 	apply: abstract func ~yuv (action: Func (ColorYuv))
 	apply: abstract func ~monochrome (action: Func (ColorMonochrome))
 	resizeTo: override func (size: IntVector2D) -> Image {
-		result : Image
-//	TODO: Actually resize the image
```
Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: At last it's gone.

**Comment by davidpiuva** [comment]
@fredrikbryntesson Peer review?

File: source/draw/RasterImage.ooc [hunk 2]
```diff
@@ -22,33 +22,10 @@ RasterImage: abstract class extends Image {
...
-		mappingTransformInverse := mappingTransform inverse
-		upperLeft = mappingTransformInverse * source leftTop
-		upperRight = mappingTransformInverse * source rightTop
-		downLeft = mappingTransformInverse * source leftBottom
-		downRight = mappingTransformInverse * source rightBottom
-		this copy(size toFloatVector2D(), source, FloatPoint2D new(), FloatPoint2D new(), FloatPoint2D new())
-	}
-	copy: func ~fromMoreParams (size: FloatVector2D, source: FloatBox2D, upperLeft, upperRight, lowerLeft: FloatPoint2D) -> This {
-		result := RasterRgba new(size ceiling() toIntVector2D())
-//		TODO: The stuff
```
Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: Good bye, old friend.

**Comment by marcusnaslund** [comment]
This fixes #574 - nice.

**Pull request merged**